src/main.c

```diff
@@ -37,6 +39,71 @@ void print_help(char *argv[]) {
   printf("\tApplication is started when called without argument or after firmware load\n\n");
 }
 
+#include <string.h>
+#define USB_TIMEOUT 5000
```

Comment:
If we add USB_TIMEOUT then this should be used everywhere a timeout value is needed